tests: Bluetooth: Tester: Use new conf file for netcore for audio

Thalley opened the pull request:
Instead of using the hci_ipc sample and then applying an overlay on top of that, we create a new configuration file specifically for the network core for audio
testing.

The purpose to simply the build setup, keep the configratution in the application itself, as well as being able to more easily apply this configuration.

Thalley committed:
tests: Bluetooth: Tester: Use new conf file for netcore for audio

Instead of using the hci_ipc sample and then applying an
overlay on top of that, we create a new configuration
file specifically for the network core for audio
testing.

The purpose to simply the build setup, keep the configratution
in the application itself, as well as being able to more
easily apply this configuration.

The LE audio overlays are automatically applied for the
nrf5340 Audio DK, but needs to be manually applied
with -DCONFIG_TESTER_AUDIO for other boards.

Signed-off-by: Emil Gydesen <emil.gydesen@nordicsemi.no>

Thalley commented:
#AutoPTS run zephyr nrf53 BAP/UCL/STR/BV-523-C

codecoup-tester commented:
Scheduled PR https://github.com/zephyrproject-rtos/zephyr/pull/87456#issuecomment-2748127184, board: nrf53, estimated start time: 14:27:54, test case count: 1, estimated duration: 0:11:12<details><summary>Test cases to be run</summary>BAP/UCL/STR/BV-523-C</details>

codecoup-tester commented:
AutoPTS Bot results:
No failed test found.
<details><summary>Successful tests (1)</summary>BAP     BAP/UCL/STR/BV-523-C            PASS</details>

Thalley commented:
#AutoPTS run zephyr nrf53 BAP/UCL/STR/BV-523-C

codecoup-tester commented:
Scheduled PR https://github.com/zephyrproject-rtos/zephyr/pull/87456#issuecomment-2778775456, board: nrf53, estimated start time: 15:47:40, test case count: 1, estimated duration: 0:11:11<details><summary>Test cases to be run</summary>BAP/UCL/STR/BV-523-C</details>

codecoup-tester commented:
AutoPTS Bot results:
<details><summary>Failed tests (1)</summary>BAP     BAP/UCL/STR/BV-523-C            BTP ERROR</details>

Thalley committed:
tests: Bluetooth: Tester: Add new controller audio config

Adds a new controller audio configuration that can be applied
to multiple boards and not specific to the nRF5340 cpunet
as the existing setup uses from hci_ipc.

Signed-off-by: Emil Gydesen <emil.gydesen@nordicsemi.no>

Thalley commented:
Closed in favor of https://github.com/zephyrproject-rtos/zephyr/pull/88300
